flow/frame_timings.cc

```diff
@@ -255,7 +255,9 @@ const char* FrameTimingsRecorder::GetFrameNumberTraceArg() const {
 }
 
 void FrameTimingsRecorder::AssertInState(State state) const {
-  FML_DCHECK(state_ == state);
+  FML_DCHECK(state_ == state)
+      << "Expected state " << static_cast<uint32_t>(state) << ", actual state "
+      << static_cast<uint32_t>(state_);
```

Comment:
nit: Consider a (debug-only) operator to convert the state to a friendly string.

Reply:
Done.